test/index.ts

```diff
@@ -497,6 +497,15 @@ describe('Storage', () => {
     });
 
     describe('storage classes', () => {
+      it('should expand metadata.archive', done => {
```

Comment:
Is there a way to de-dupe the logic that's repeated for every storage class here? I'm far from a Node expert so feel free to tell me that this is impossible or a bad idea! :)

Reply:
Not at all! I think that's a pretty good catch, however because this is our test suite, I usually err on the side of over-emphasis. Having each case handled in its own test makes it stand out as more deliberate than going through a loop or trying to group them all in a single test. Overall, optimization isn't a thought I hold too close to tests-- I think being more verbose and readable is a good thing, and hopefully holds future test writers to the same standard... at least, that's the thought anyway! :)